HospitalSystem.Core/People/Doctor/DoctorRepository.cs

```diff
@@ -1,90 +1,88 @@
 using MongoDB.Driver;
 using MongoDB.Bson;
 
-namespace HospitalSystem.Core
+namespace HospitalSystem.Core;
+public class DoctorRepository
 {
-    public class DoctorRepository
-    {
-        private MongoClient _dbClient;
-
-        public DoctorRepository(MongoClient _dbClient)
-        {
-            this._dbClient = _dbClient;
-        }
+    private MongoClient _dbClient;
 
-        public IMongoCollection<Doctor> GetAll()
-        {
-            return _dbClient.GetDatabase("hospital").GetCollection<Doctor>("doctors");
-        }
-
-        public void Upsert(Doctor newDoctor)
-        {
-            var doctors = GetAll();
-            doctors.ReplaceOne(doctor => doctor.Id == newDoctor.Id, newDoctor, new ReplaceOptions {IsUpsert = true});
-        }
+    public DoctorRepository(MongoClient _dbClient)
+    {
+        this._dbClient = _dbClient;
+    }
 
-        public List<Doctor> GetManyBySpecialty(Specialty specialty)
-        {
-            var doctors = GetAll();
-                var specizedDoctors =
-                    from doctor in doctors.AsQueryable<Doctor>()
-                    where doctor.Specialty == specialty
-                    select doctor;
+    public IMongoCollection<Doctor> GetAll()
+    {
+        return _dbClient.GetDatabase("hospital").GetCollection<Doctor>("doctors");
+    }
```

Comment:
Maybe don't return a public MongoCollection

Reply:
dont know what else, copied it from you at the begining

Reply:
eh, changes my whole repo. It works fine